src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -76,7 +76,7 @@ internal static bool DoSslHandshake(SafeSslHandle context, IntPtr recvPtr, int r
 
                 if ((retVal != -1) || (error != libssl.SslErrorCode.SSL_ERROR_WANT_READ))
                 {
-                    throw CreateSslException(context, "SSL Handshake failed: ", retVal);
+                    Debug.Fail("SSL Handshake failed with error:" + error);
```

Comment:
What happens here for release?  We still want to proceed to do all of the stuff below?